app/assets/javascripts/app/views.js

```diff
@@ -65,7 +65,8 @@ app.views.Base = Backbone.View.extend({
     var self = this;
     _.each(this.subviews, function(property, selector){
       var view = _.isFunction(self[property]) ? self[property]() : self[property];
-      if(view) {
+      if (view && self.$(selector).length > 0) {
+        self.$(selector).first().empty();
```

Comment:
Shouldn't `.html(...)` in the next line replace the old contents with new ones? Why do we have to empty it first?